src/rules/enforce-boolean-naming-prefixes.ts

```
@@ -0,0 +1,563 @@
+        'Boolean {{type}} "{{name}}" should start with an approved prefix: {{prefixes}}',
+    },
+  },
+  defaultOptions: [{ prefixes: DEFAULT_BOOLEAN_PREFIXES }],
+  create(context, [options]) {
+    const approvedPrefixes = options.prefixes || DEFAULT_BOOLEAN_PREFIXES;
+
+    /**
+     * Check if a name starts with any of the approved prefixes
+     */
+    function hasApprovedPrefix(name: string): boolean {
+      return approvedPrefixes.some(prefix =>
```

ESLint

[error] Replace `prefix` with `(prefix)`

(prettier/prettier)